src/app/players/playerpoints.service.ts

```diff
@@ -0,0 +1,51 @@
+import { Injectable } from '@angular/core';
+import { AngularFirestore, AngularFirestoreCollection } from '@angular/fire/firestore';
+import { map } from 'rxjs/operators';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class PlayerPointsService {
+
+  playerPointsCollection: AngularFirestoreCollection<PlayerPoint>;
+
+  constructor(private afs: AngularFirestore) {
+    this.playerPointsCollection = this.afs.collection('playerPoints');
+  }
+
```

Comment:
you'll need another method here to query all points of all team members using `where('teamId', '==', teamId)` instead of playerId